CSS/4-about.css

```diff
@@ -0,0 +1,115 @@
+/* ABOUT */
+
+.about {
+    background-color: #1c1a19;
+    background-image: url(../img/bg/Frame.png);
+    background-repeat: no-repeat;
+    background-position: 100% 10%;
+    background-size: 40%;
+    display: grid;
+    grid-template-columns: 10% 40% 40% 10%;
+    grid-template-rows: 1fr 10% 1fr;
+  }
```

Comment:
Please stick to Figma, please reduce the margin-top

<img width="872" alt="image" src="https://user-images.githubusercontent.com/7243132/170837032-b415bb47-8768-4ea8-9ae7-3f8278017412.png">